Explorer/Assets/Scripts/Global/Static/StaticSceneLauncher.cs

```diff
@@ -1,206 +0,0 @@
-﻿using Cysharp.Threading.Tasks;
-using DCL.AssetsProvision;
-using DCL.Browser;
-using DCL.Browser.DecentralandUrls;
-using DCL.DebugUtilities;
-using DCL.Diagnostics;
-using DCL.Multiplayer.Connections.DecentralandUrls;
-using DCL.Multiplayer.Connections.Messaging.Hubs;
-using DCL.Multiplayer.Connections.RoomHubs;
-using DCL.PluginSystem;
-using DCL.Profiles;
-using DCL.Web3;
-using DCL.Web3.Authenticators;
-using DCL.Web3.Identities;
-using DCL.WebRequests;
-using MVC;
-using SceneRunner.Scene;
-using System;
-using System.Collections.Generic;
-using System.Threading;
-using DCL.PerformanceAndDiagnostics.DotNetLogging;
-using DCL.Utilities.Extensions;
-using DCL.Web3.Accounts.Factory;
-using Global.AppArgs;
-using Global.Dynamic;
-using UnityEngine;
-
-namespace Global.Static
-{
-    /// <summary>
-    ///     An entry point to install and resolve all dependencies
-    /// </summary>
-    public class StaticSceneLauncher : MonoBehaviour
```

Comment:
I have removed the static scene launcher. I think it is deprecated and not used anywhere else, but i would need a confirmation